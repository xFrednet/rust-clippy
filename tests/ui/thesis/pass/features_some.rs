//@rustc-env: CLIPPY_PETS_PRINT=1
#![feature(lint_reasons)]

#[warn(clippy::borrow_pats)]
fn main() {}
