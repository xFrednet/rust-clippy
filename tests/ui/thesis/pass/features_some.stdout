Disabled due to feature use
